components/layout/HeroBanner.tsx

```diff
@@ -1,175 +1,49 @@
-          <Box sx={{ flex: 1, order: { xs: '2', md: '1' } }}>
-            <Box className="subheading" sx={{ mt: 2, mb: 4 }}>
-              <MuiMarkdown
-                overrides={{
-                  p: {
-                    component: Typography,
-                    props: {
-                      variant: 'body1',
-                      gutterBottom: true,
-                    } as TypographyProps,
-                  },
-
-                  span: {
-                    component: Typography,
-                    props: { variant: 'body1' } as TypographyProps,
-
-                  },
-                  a: {
-                    props: {
-                      style: {
-                        color: '#FFD200',
-                        textDecoration: 'underline',
-                      },
-                    },
-                  },
-                }}
-              >
-                {`${subheading}`}
-
-              </MuiMarkdown>
-
-            </Box>
-            {subheading2 && (
-            <Box className="subheading-secondary" sx={{ mb: 4 }}>
-              <MuiMarkdown
-                overrides={{
-                  p: {
-                    component: Typography,
-                    props: {
-                      variant: 'body1',
-                      gutterBottom: true,
-                    } as TypographyProps,
-                  },
-
-                  span: {
-                    component: Typography,
-                    props: { variant: 'body1' } as TypographyProps,
-
-                  },
-                }}
-              >
-                {` ${subheading2}`}
-
-              </MuiMarkdown>
-
-            </Box>
-            )}
-            {ctaText && ctaLink && (
-              <Button
-                variant="contained"
-                color="primary"
-                href={ctaLink}
-                data-href={ctaLink}
-                aria-label={ariaLabels?.ctaButton}
-                onClick={handleLinkClick}
-                sx={{
-                  px: 4,
-                  '&:hover': {
-                    color: 'primary.dark',
-                    backgroundColor: 'secondary.main',
-                  },
-                }}
-              >
-                {ctaText}
-              </Button>
-            )}
-          </Box>
-          {(imgsrc || children) && (
-            <Box
-              className="hero-additionals"
-              sx={getHeroAdditionalsStyles}
-            >
-              {imgsrc && (
-                <ImageContainer
-                  src={imgsrc}
-                  width={513}
-                  height={386}
-                  alt=""
-                  style={{ maxWidth: '100%' }}
-                  useImageDimensions
-                  priority={priority}
-                />
-              )}
-              {children}
-            </Box>
-          )}
-        </Box>
-      </SectionContainer>
+          {header}
+        </Typography>
+        <EligibilityButton />
```

Comment:
Only the home page will be having this button, maybe we can go back to passing in cta as optional and then rendering only if we need it.

Reply:
Fixed